File: Data Structures & Algorithms/reverse-bits/submission-0.cpp
class Solution {
public:
    uint32_t reverseBits(uint32_t n) {
        uint32_t res = 0;
        uint32_t a = n;

        for(int i = 0; i < 32; i++)
        {
            res = res << 1 ;
            res |= (a & 1);
            a >>= 1;
        }
        return res;
    }
};

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        length = len(nums) // 3

        elements = {}

        for n in nums:
            elements[n] = elements.get(n, 0) + 1
        
        res = []
        for n in elements:
            if elements[n] > length:
                res.append(n)
        
        return res